compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangNodeTransformer.java

```diff
@@ -4443,6 +4453,17 @@ private BLangIdentifier createIdentifier(Location pos, String value, Set<Whitesp
         return bLIdentifer;
     }
 
+    private BLangLiteral createSimpleLiteral(TemplateExpressionNode expressionNode) {
```

Comment:
Shall we rename this to something like
```suggestion
    private BLangLiteral createEmptyStringLiteral(TemplateExpressionNode expressionNode) {
```